tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -1228,20 +1236,22 @@ describe('IssueList,', function() {
       expect(fetchDataMock).toHaveBeenCalled();
     });
 
-    it('fetches data on location change', function() {
+    it('fetches data on location change', async function() {
       const queryAttrs = ['query', 'sort', 'statsPeriod', 'cursor', 'groupStatsPeriod'];
-      let location = cloneDeep(props.location);
-      queryAttrs.forEach(async (attr, i) => {
```

Comment:
this test may have executed a little weirdly before because we have an async function within the forEach loop, but the test function itself is not async so I'm guessing that the contents of the forEach loop were running during other tests possibly creating side effects. This was causing some of my later tests to fail unexpectedly.